tools/components/components.go

```diff
@@ -0,0 +1,209 @@
+package components
```

Comment:
are components.go and constants.go only utilized by files in the 'util' subdirectory? If so, should they be in their own directory or inside util as well?

Reply:
just want to keep separated from other operator code and putting inside tools package which is only generating CSV and get digest and has no dependencies other operators code, So in case if we need to remove in future considering the fetching digest will be a fully automated solution -as per requirements- https://projects.engineering.redhat.com/browse/OSBS-8294